### DISCUSSION TITLE
How to get Open Sourcerer and Heart On Your Sleeve?

### DISCUSSION THREAD

**Discussion by cedev935** [Schweinepriester/github-profile-achievements#831] [Q&A]
This is my GitHub:
https://github.com/cedev935
I want to receive Open Sourcerer and Heart On Your Sleeve.
Please help me.

**Comment by olehrab** [comment]
'Open Sourcerer' achievement badge is not released to public GitHub users. You can follow any updated on the official blog of GitHub [here](https://github.blog/).

When they launched the first set of achievement badges, they previously also posted a blog over [here](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/) to let users know more information. Hence, they might do the same thing (i.e. post a new blog) to announce updates to the achievement system on GitHub.

Other than that, there has been no information on how to achieve this badge.

Hope this clarifies & answers your question.

**Comment by maksymhu** [comment]
To earn the Open Sourcerer achievement on GitHub, you need to make a qualifying contribution to an open source project on GitHub. Here are the steps you can follow:

Sign up for a GitHub account if you haven't already.
Find an open source project on GitHub that you would like to contribute to. You can search for open source projects using GitHub's Explore feature, or you can look for projects in specific programming languages or topics that interest you.
Fork the repository of the open source project that you want to contribute to. This creates a copy of the project in your own GitHub account that you can make changes to.
Make a qualifying contribution to the project. This can be anything from fixing a bug or adding a new feature, to improving documentation or submitting a new translation. Your contribution must be accepted and merged into the main repository by the project maintainers to count towards the Open Sourcerer achievement.
Once your contribution has been merged, you should receive a notification from GitHub that your contribution has been accepted. After 30 days of making your first contribution, you will earn the Open Sourcerer badge on your GitHub profile.
Note that GitHub has specific criteria for what constitutes a qualifying contribution, so be sure to review the contribution guidelines for the project you want to contribute to before getting started. Good luck!
